dagfactory/dagbuilder.py

```diff
@@ -2,11 +2,25 @@
 from datetime import timedelta, datetime
 from typing import Any, Callable, Dict, List, Union
 
+import os
+
 from airflow import DAG, configuration
+from airflow.models import Variable
+
+# kubernetes operator
+from airflow.kubernetes.secret import Secret
```

Comment:
I think these imports should be `airflow.contrib.kubernetes` to keep it backwards compatible with earlier version of Airflow.